### PR TITLE
LPS-104105 This should not be necessary

### PR DESCRIPTION
**Pull request by jkappler**

### CODE REVIEW

**Comment by jkappler** [comment]
@ealonso, @adolfopa

**Comment by brianchandotcom** [comment]
@jkappler 1.) there is a rebase error where

com.liferay.document.library.kernel.service.persistence.DLFolderFinder.countFE_ByG_F

is showing up twice in default.xml

2.) Rename _FET to _FETI (better matches existing patterns)

3.) For your new methods, go with:

folderId, mimeTypes, fileEntryId, includeMountFolders, status, start, end

Basically, I want to match service.xml.

includeMountFolders matches the hidden_ field in DLFolder.

In a separate pull (after this is merged in), reorder the methods in *DLFolderServiceImpl to follow this order. So this is wrong:

long groupId, long folderId, int status, String[] mimeTypes

long groupId, long folderId, int status, String[] mimeTypes, boolean includeMountFolders

Basically, move status, mimeTypes, and includeMountFolders to follow the new order I set for you for the new methods you're adding. I'm fine with even not deprecating the old methods since this is 7.3 only.

Thx!
